interface/src/app/node/node.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * "adage.gene.network" module.
+ */
+
+angular.module('adage.node', [
+  'ui.router',
+  'placeholders',
```

Comment:
The `angular-placeholders` module is handy when mocking up pages (http://joshdmiller.github.io/angular-placeholders/ ) but I don't see any place you're using them here so this dependency can be removed.